clinvar_ingest/api/main_test.py

```diff
@@ -1,9 +1,20 @@
+import json
+import logging.config
+
+import pytest
 from fastapi.testclient import TestClient
 
 from clinvar_ingest.api.main import app
 
 
-def test_status_check(caplog) -> None:
+@pytest.fixture
+def log_conf():
```

Comment:
Our tests require the logging to be configured, so this just initializes it within the tests rather than within the FastAPI app itself.